src/app/config/store.js

```diff
@@ -16,6 +17,7 @@ export const store = createStore(
     combineReducers({
         state: reducer,
         user: userReducer,
+        newTeam: newTeamReducer,
```

Comment:
`teams` ?

Reply:
but still I am not sure if you need creating new reducer at all.

Reply:
Happy to change the name on this as I am not too happy with 'newTeam' either but couldn't think of a more appropriate name at the time. Since PRing this it has changed a lot, as it used to contain `usersInTeam` `usersNotInTeam` and `allPreviewUsers`, but now it is just `allPreviewUsers` and `unsavedChanges`. So this could contain the upcoming 'edit' team data and list of teams data too.

Aware there is state.teams but this is cumbersome and that datastore is more tailored to what Zebedee returns rather than the new system Cognito for user and team management. - Will change the name to 'teams'. Although allPreviewUsers may not fit so nicely as they are not 'in' the team yet - but `allPreviewUsers` is used after being populated to list all possible users an admin could choose to add to a new team, then that data field is used to keep track of users that are in a basket ready to be added to the new team, as if they aren't in that field then they are in a basket to be added to the team.